nixos/modules/services/misc/nix-optimise.nix

```
@@ -15,7 +15,7 @@ in
 
       dates = lib.mkOption {
         default = [ "03:45" ];
-        type = with lib.types; listOf str;
+        type = with lib.types; either singleLineStr (listOf str);
         description = ''
```

Same as above, can we do something like

```suggestion
        apply = lib.toList;
        description = ''
```